lib/utils/forkedblockchain.js

```diff
@@ -122,30 +116,20 @@ ForkedBlockchain.prototype.createGenesisBlock = function(callback) {
   });
 };
 
-ForkedBlockchain.prototype.createForkedStorageTrie = function(address) {
-  address = to.hex(address);
-
-  var trie = new ForkedStorageTrie(this.data.trie_db, null, {
-    address: address,
-    stateTrie: this.stateTrie,
-    blockchain: this,
-    fork: this.fork,
-    forkBlockNumber: this.forkBlockNumber
-  });
-
-  this.storageTrieCache[address] = trie;
-
-  return trie;
-};
-
-ForkedBlockchain.prototype.lookupStorageTrie = function(address, callback) {
```

Comment:
Reason for change: we have a better one now. And the cache is mostly unnecessary as it complicates things (like when the stateTrie root is from a  previous block but the cache contains data from a newer block) and the cache is already properly handled in the VM, which most of the calls to this fn come from.